pubspec.yaml

```diff
@@ -27,14 +27,17 @@ dependencies:
   json_annotation: ^4.8.1
   loggy: ^2.0.3
   shimmer: ^3.0.0
+  widgetbook: ^3.7.1
+  widgetbook_annotation: ^3.1.0
 
 dev_dependencies:
   auto_route_generator: ^7.3.2
-  build_runner:
+  build_runner: ^2.4.9
```

Comment:
Did you "had" to declare the version explicitly?

In the auto_route documentation, the `build_runner` version is let loose. I guess this is for avoid dependencing version conflicts